src/core/Akka.Streams.Tests/Dsl/ActorRefSourceSpec.cs

```diff
@@ -178,23 +163,15 @@ public void A_ActorRefSource_must_not_buffer_elements_after_receiving_Status_Suc
         }
 
         [Fact]
-        public void A_ActorRefSource_must_after_receiving_Status_Success_allow_for_earlier_completion_with_PoisonPill()
+        public void A_ActorRefSource_must_complete_and_materialize_the_stream_after_receiving_Status_Success()
         {
             this.AssertAllStagesStopped(() =>
             {
-                var s = this.CreateManualSubscriberProbe<int>();
-                var actorRef = Source.ActorRef<int>(3, OverflowStrategy.DropBuffer)
-                    .To(Sink.FromSubscriber(s))
+                var (actorRef, done) = Source.ActorRef<int>(3, OverflowStrategy.DropBuffer)
+                    .ToMaterialized(Sink.Ignore<int>(), Keep.Both)
                     .Run(Materializer);
-                var sub = s.ExpectSubscription();
-                actorRef.Tell(1);
-                actorRef.Tell(2);
-                actorRef.Tell(3);
                 actorRef.Tell(new Status.Success("ok"));
-                sub.Request(2); // not all elements drained yet
-                s.ExpectNext(1, 2);
-                actorRef.Tell(PoisonPill.Instance);
-                s.ExpectComplete(); // element `3` not signaled
+                done.ContinueWith(_ => Done.Instance).Result.Should().Be(Done.Instance);
```

Comment:
`Sink.Ignore` is another opertator with the wrong signature: it should return `Task<Done>` not just `Task`